### DISCUSSION TITLE
How to celebrate the 10 years?

### DISCUSSION THREAD

**Discussion by marienfressinaud** [FreshRSS/FreshRSS#4294] [Ideas]
FreshRSS will be 10 years old in October (cf. https://github.com/FreshRSS/FreshRSS/commit/fb57be5a5af3a2fb46b2dbf2b503ffe78eb5cf49), I’d like to do a little something to celebrate this date, but I'm afraid to start something too big for me. I was thinking to update the website (https://freshrss.org), keeping it simple but making make it more appealing and adding some missing information (a link to the demo and to some existing FreshRSS instances for instance).

Do you have any special desires to celebrate these 10 years?

**Comment by oupala** [comment]
I would propose 2 different things:

- move from GitHub to an opensource forge (while possibly mirroring to GitHub)
- organize a small human event in a bar for those who are close from Grenoble

**Reply by marienfressinaud** [reply]
Moving from GitHub is probably more challenging: we would have to choose a tool (Gitlab / Framagit?) import all the repositories, adapt the CI scripts, change the links everywhere… and I probably forget several other things. I understand the reasons to move, but I would suggest to open a different discussion so it's not tied to the anniversary date.

A small event at Grenoble is totally doable though! I prefer to do it outside of a bar (noisy, and some people might not be at ease in such place), but I can contact La Turbine (https://turbine.coop/), they often host events related to free software and communities :)

**Reply by oupala** [reply]
What about talking while hiking in the mountain? It would an original way to celebrate this 10 years anniversary (it's not exclusive with an event in town).

**Reply by math-GH** [reply]
Short feedback from my side to the Github/Non-Github discussion: 
I became an active member of FreshRSS because of the easy access via GitHub.

**Reply by oupala** [reply]
Of course @math-GH, I know that GitHub is important for visibility. That's why I was also proposing mirroring to GitHub.

**Comment by qwertygc** [comment]
Congratulations! An idea would be to put some kind of "guestbook" or something like that on the site, so that FreshRSS users can testify about the use of this software, sometimes for a long time (I've been using freshRSS for so long ?)

**Reply by marienfressinaud** [reply]
This can be done indeed! We got a lot of great feedback over the years, it may be time to give them a place :)

**Comment by Alkarex** [comment]
Maybe some kind of online event, e.g. on Framateam? Potentially with video :-)

**Reply by marienfressinaud** [reply]
It could be great, but I'm less interested to organize it myself :sweat_smile: Do you (or other people) want to organize it?

**Comment by marienfressinaud** [comment]
I started a new website (in a [new branch](https://github.com/FreshRSS/freshrss.org/tree/new-website)). Here's how it looks for now (illustrations come from [undraw.co](https://undraw.co/illustrations)):

![Screenshot 2022-07-12 at 10-41-55 FreshRSS a free self-hostable feeds aggregator](https://user-images.githubusercontent.com/1436309/178449041-26bb85b7-bb28-42cb-803e-a32373aac41b.png)

**Reply by Frenzie** [reply]
Looks cool! :-)

**Reply by marienfressinaud** [reply]
I opened a PR to track the things I have to do before getting done: https://github.com/FreshRSS/freshrss.org/pull/50

**Comment by Offerel** [comment]
Wow, 10 years !!! I really love this little piece of software. Frankly, I have no idea if I could live without that thing. Since Grenoble is a little bit far away from Munich/Bavaria, it would be nice if there could be some sort of live stream or interactive come-together. Would be nice.

BTW: I have changed all my personal GitHub repository's from GitHub to Codeberg.org. It is not, that i didn't trust GitHub anymore, but I prefer to use a more open sourced view. But this up to the developers and is not an easy switch, because not all comfort functions are available there. Additionally, the userbase is small compared to GitHub.

**Reply by oupala** [reply]
In France, we also have [Framagit](https://framagit.org/) which is hosted by the well known Framasoft association. It seems to be similar to Codeberg except that Framagit is running GitLab and not Gitea.

**Comment by math-GH** [comment]
The release of 1.20.0 would be a nice 10years anniversary gift, isn't it?

**Reply by Alkarex** [reply]
Working on it!

**Reply by Alkarex** [reply]
https://github.com/FreshRSS/FreshRSS/discussions/4517

**Comment by marienfressinaud** [comment]
**The new website is almost finished (https://next.freshrss.org/).** I still have to update two links on the "10 years" page (cf. [my todo list](https://github.com/FreshRSS/freshrss.org/pull/50)). You have 10 more days if you want to review the website ;) I didn't create a dedicated page for the testimonials, but I think we'll get more once the anniversary will be announced so I think I'll do it in October or November.

I'll publish the website and start communicating on social networks on Saturday 1st October. I also plan to publish a news in French on [linuxfr.org](https://linuxfr.org) (and probably on my own website) few days later.

On Thursday 20th October, I organize an event at [La Turbine](https://turbine.coop/) at Grenoble (France). I'll do a [Contribatelier](https://contribateliers.org/) (i.e. a workshop to contribute to free softwares, not only to the code) and bring cakes :)  I'll share more information later.

**Reply by Frenzie** [reply]
I'm afraid I'll probably have to skip. I'm currently using up most of my spare free days due to moving and a few days later I'll have several extremely long days in Brussels. Even with the TGV it's about 7 hours. (There's also a 1.5 hour flight; of course that's still closer to 5-6 hours total with getting to and from airports etc. so it's a toss up in that regard really.) But have fun!

**Reply by Alkarex** [reply]
I will not be able to attend physically, but if there is a timeslot with a videolink, I will happily say hello :-)
Likewise, I suggest an additional online meetup, either before, during, or after the physical event. What do you think? I can make a [Framadate ](https://framadate.org/) to find a good timeslot with at least the main contributors.

**Reply by aledeg** [reply]
I won't attend either. There is an ocean between us :)

**Reply by Frenzie** [reply]
A videocall should definitely be doable.

**Reply by marienfressinaud** [reply]
Let's do this :)
